themes/default/routes.js

```diff
@@ -19,7 +19,7 @@ class Routes extends Component {
           path={data.url}
           render={({ staticContext }) => (
             <Page
-              route={staticContext || data}
+              route={staticContext ? JSON.parse(staticContext) : data}
```

Comment:
Huh? This will break things. `staticContext` is already a JSON object, trying to parse it again will throw an error.

Reply:
Yeah not sure why I did that...fixed.